src/vdaf/poplar1.rs

```diff
@@ -357,7 +358,7 @@ where
     P: Prg<L>,
 {
     #[allow(clippy::many_single_char_names)]
-    fn shard(&self, input: &IdpfInput) -> Result<Vec<Poplar1InputShare<I, L>>, VdafError> {
+    fn shard(&self, input: &IdpfInput) -> Result<((), Vec<Poplar1InputShare<I, L>>), VdafError> {
```

Comment:
The API would have been clearer if `()` had been `PublicShare`.

Reply:
Good point! https://github.com/divviup/libprio-rs/pull/371